schema.graphql

```diff
@@ -186,9 +209,11 @@ type User @entity {
   donations: [Donation!]! @derivedFrom(field: "user")
   earlyLiquidityPurchases: [EarlyLiquidityPurchase!]!
     @derivedFrom(field: "user")
+  activities: [Activity!]! @derivedFrom(field: "user")
   totalUSDCEffect: BigInt!
   nonceSeperator: BigInt!
   totalImpactPoints: BigInt!
+  totalStakedGlow: BigInt!
```

Comment:
The totalStakedGlow property changes when the stake or unstaked event handlers are invoked. but if the user already has staked glow then the original amount will be 0 in the User table @0xSimbo